README.md

```diff
@@ -642,6 +642,8 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
         - sink_taxid.csv: csv TAXID table single sample, directly from https://github.com/maxibor/sourcepredict/blob/bcaf000c3d91d0aaf9fbe25c7218348174a2e0b3/data/test/dog_test_sink_sample.csv
         - sources_taxid.csv: csv TAXID table of 20 samples, subsampled from https://github.com/maxibor/sourcepredict/blob/bcaf000c3d91d0aaf9fbe25c7218348174a2e0b3/data/modern_gut_microbiomes_sources.csv
         - sources_labels.csv: csv table with sample ids and sample source, subsampled from https://github.com/maxibor/sourcepredict/blob/bcaf000c3d91d0aaf9fbe25c7218348174a2e0b3/data/modern_gut_microbiomes_labels.csv
+        - taxa_sqlite.xz: taxa.sqlite database file downloaded with ete3, compressed with xz
```

Comment:
What exact version of ete3? And then you're good to go :) And again clever with the xz!